src/main/io/serial_cli.c

```diff
@@ -544,6 +544,7 @@ const clivalue_t valueTable[] = {
 #ifdef TELEMETRY
     { "telemetry_switch",           VAR_UINT8  | MASTER_VALUE | MODE_LOOKUP,  &masterConfig.telemetryConfig.telemetry_switch, .config.lookup = { TABLE_OFF_ON } },
     { "telemetry_inversion",        VAR_UINT8  | MASTER_VALUE | MODE_LOOKUP,  &masterConfig.telemetryConfig.telemetry_inversion, .config.lookup = { TABLE_OFF_ON } },
+    { "telemetry_flvss_cells",      VAR_UINT8  | MASTER_VALUE | MODE_LOOKUP,  &masterConfig.telemetryConfig.telemetry_flvss_cells, .config.lookup = { TABLE_OFF_ON } },
```

Comment:
not sure if the name of this is correct, since it only applies to frsky telemtry it should probably be something like 'frsky_send_cell_voltages`.